custom_components/e3dc_rscp/coordinator.py

```diff
@@ -62,6 +69,40 @@ async def async_connect(self):
         except Exception as ex:
             raise ConfigEntryAuthFailed from ex
 
+        # get the additional powermeters and re-create
```

Comment:
Please move this auto-configuration (lines 72-92) into its own private function to keep the async_connect slim.

Reply:
Done